documentation/book/con-cluster-operator-namespaces.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// assembly-operators-cluster-operator.adoc
+
+[id='con-cluster-operator-namespaces-{context}']
+= Watching multiple namespaces
+
+The Cluster Operator is able to watch for the `Kafka`, `KafkaConnect` and `KafkaConnectS2I` resources in multiple namespaces.
```

Comment:
The Cluster Operator is able to watch for the `Kafka`, `KafkaConnect` and `KafkaConnectS2I` resources in multiple namespaces.
It does not have to be installed in every single namespace separately.
The only reason to run multiple Cluster Operators in a single cluster is to use different versions of {ProductName}.

TO

The Cluster Operator is able to watch for `Kafka`, `KafkaConnect`, and `KafkaConnectS2I` resources in multiple namespaces without being installed on each namespace individually.

NOTE: The only reason to run multiple Cluster Operators in a single cluster is to use different versions of {ProductName}.